automation/taskcluster/update_android_components.sh

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# If a command fails then do not proceed and fail this script too.
+set -ex
+
+# Install dependencies (TODO: Move to Docker image)
```

Comment:
nit:Should we open an issue to keep track of it? :)

Reply:
👍